From af4d0f5706740848c3fef67a7ea0b78cf31ddb90 Mon Sep 17 00:00:00 2001
From: Stefano Lattarini <stefano.lattarini@gmail.com>
Date: Thu, 30 Aug 2012 18:53:18 +0200
Subject: [PATCH 15/89] tests: remove unused AM_TESTS_ENVIRONMENT entries

Probably a copy&paste errors, likely from the TESTS_ENVIRONMENT
definition in the 'tests/Makefile.am' in GNU coreutils.

* tests/Makefile.am (AM_TESTS_ENVIRONMENT): Don't define nor export
any of the variables PREFERABLY_POSIX_SHELL, REPLACE_GETCWD, CC,
MAKE, PERL.

Copyright-paperwork-exempt: yes
---
 tests/Makefile.am | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 33e8f8e..c97400b 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -107,16 +107,11 @@ AM_TESTS_ENVIRONMENT =				\
   srcdir='$(srcdir)'				\
   top_srcdir='$(top_srcdir)'			\
   AWK='$(AWK)'					\
-  CC='$(CC)'					\
-  MAKE=$(MAKE)					\
   PACKAGE_BUGREPORT='$(PACKAGE_BUGREPORT)'	\
   PACKAGE_VERSION=$(PACKAGE_VERSION)		\
   CONFIG_HEADER='$(abs_top_builddir)/lib/config.h' \
   ENABLE_DEVICE_MAPPER=$(ENABLE_DEVICE_MAPPER)	\
   PARTED_TEST_NAME=`basename '$(abs_srcdir)'`,`echo $$tst|sed 's,^\./,,;s,/,-,g'`\
-  PERL='$(PERL)'				\
-  PREFERABLY_POSIX_SHELL='$(PREFERABLY_POSIX_SHELL)' \
-  REPLACE_GETCWD=$(REPLACE_GETCWD)		\
   PATH='$(abs_top_builddir)/parted$(PATH_SEPARATOR)'"$$PATH" \
   VERSION=$(VERSION)				\
   ; 9>&2
-- 
1.8.5.3

